lang/levels/classic/text_fre.po

```diff
@@ -0,0 +1,537 @@
+
+#. Vassago (lv302)
+#: guitext:19
+msgctxt "In-game message"
+msgid "You have triggered an alarm Keeper. More Heroes are on their way! "
+msgstr "Vous avez déclenché une alarme Gardien. D'autres héros sont en route ! "
+
+#. Vassago (lv302)
+#: guitext:20
+msgctxt "In-game message"
+msgid "You have more attacks to endure Keeper. Can you survive these relentless attacks?"
+msgstr "Vous avez plus d'attaques à endurer Gardien. Pouvez-vous survivre à ces attaques incessantes ?"
+
+#. Vassago (lv302)
+#: guitext:21
+msgctxt "In-game message"
+msgid "Yet more Hero reinforcements are on their way! But I think this is their last desperate attack. Survive this and victory may be yours!"
+msgstr "D'autres renforts de héros sont en route ! Mais je pense que c'est leur dernière attaque désespérée. Survivez à cela et la victoire sera peut-être à vous !"
+
+#. Vassago (lv302)
+#: guitext:22
+msgctxt "In-game message"
+msgid "Your tenacity has won you a remarkable victory here Keeper. Legends will tell of your achievements here today!"
+msgstr "Votre ténacité vous a valu une remarquable victoire ici Gardien. Les légendes raconteront vos exploits de ce jour !"
+
+#. Vassago (lv302)
+#: guitext:23
+msgctxt "In-game message"
+msgid "The hounds have been released!"
+msgstr "Les chiens ont été libérés !"
+
+#. Midway (lv336)
+#: guitext:24
+msgctxt "In-game message"
+msgid "Souls of ancient keepers lurk to the south-west. They watch you intently with what is left of their magical powers!"
+msgstr "Les âmes des anciens gardiens se cachent au sud-ouest. Ils vous regardent attentivement avec ce qui reste de leurs pouvoirs magiques !"
+
+#. Midway (lv336)
+#: guitext:25
+msgctxt "In-game message"
+msgid "Welcome to the Avatar's castle! His domain lies to the North-East, while his followers lurk to the North-West."
+msgstr "Bienvenue au château de l'Avatar ! Son domaine se trouve au nord-est, tandis que ses partisans se cachent au nord-ouest."
+
+#. Midway (lv336)
+#: guitext:26
+msgctxt "In-game message"
+msgid "Beware, the Avatar is escaping even now...."
+msgstr "Attention, l'Avatar s'échappe maintenant..."
+
+#. Midway (lv336)
+#: guitext:27
+msgctxt "In-game message"
+msgid "The door to the east bears a simple plaque which reads, Emergency Exit - Keep Clear at all Times - Offenders will be shot!"
+msgstr "La porte à l'est porte une simple plaque sur laquelle on peut lire : Sortie de secours - Restez à l'écart en tout temps - Les contrevenants seront abattus !"
+
+#. Midway (lv336)
+#: guitext:28
+msgctxt "In-game message"
+msgid "The Battle of Midway is afoot. Beware Evil one, the enemy is strong and well defended. Turn their misled minions against each other, sit back and let the butchery begin."
+msgstr "La bataille de Midway est en cours. Attention maître, l'ennemi est fort et bien défendu. Retournez leurs sbires les uns contre les autres, et laissez la boucherie commencer."
```

Comment:
"Midway" peut être traduit par "Milieu" en référence au seigneur des anneaux, je suppose?

Reply:
I don't know the map but the "Battle of Midway" was a famous WW2 battle of a small force winning against a bigger force. I don't know if this is the same on this map. Maybe there is a famous battle in France where this happend. The map is also called the same in the menu

Reply:
@CaptainCosmotic that is correct in that case the name Midway should stay so : "Bataille de Midway"

Reply:
@CaptainCosmotic in that case there is no need to translate the name "Midway".